src/molecules/question/Question.tsx

```diff
@@ -150,6 +153,32 @@ export const QuestionData = ({
           )}
         />
       )
+    case UserQuestionType.Telephone:
+      return (
+        <Controller
+          name={question.id}
+          control={control}
+          defaultValue=""
+          rules={{
+            required: config?.mandatory, validate: (value) => {
+              if (value === '' && !config?.mandatory) {
+                return true
+              }
+              return isValidE164Number(value)
+            }
+          }}
+          render={({ field: { onChange, value } }) => (
+            <PhoneInputField
+              onChange={(e) => onChange(e.target.value)}
+              label={question.title}
+              id={question.id}
+              value={value}
+              mandatory={question.questionConfig?.mandatory}
+              placeholder='+447700900000'
```

Comment:
[C] This should be a parameter of PhoneInputField as well

```js
export const PhoneInputField = ({
  onChange,
  id,
  label,
  type,
  mandatory,
  value,
  initialCountry = 'us',
  hideSpaceAfterDialCode = true,
  placeholder = '+447700900000'
  ...props
}: PhoneInputFieldProps): JSX.Element => {
...

<input
  ...
  onChange={handlePhoneInputChange}
  placeholder={placeholder}
  />
```

Note: this does mean somehow you would need to pass values for these parameters and that will need through happen through the WizardForm component (the parent component).

```ts
interface UiOptions {
   Telephone?: {
       defaultCountry?: string
       hideSpaceAfterDialCode?: boolean
       placeholder?: string
   }
}

export interface QuestionDataProps {
  ...,
  ui_options?: UiOptions
}

export interface QuestionProps {
  ...,
  uiOptions?: UiOptions
}

export interface WizardFormProps {
  ...,
  uiOptions?: UiOptions
}
```

Reply:
`hideSpaceAfterDialCode` is not an option to expose, it's required for ensuring the values are correctly formatted. Formatting the input nicely is something that will come once the library author makes a fix I asked for (input onChange returns E164 formatted value separately from what it uses internally for display formatting, right now what it returns and what it displays is the same string).

The others make sense, default country, available countries (country or list of countries you want to limit the input to), and placeholder.